### PR TITLE
Fix info about 1.20.0 and add 1.21.0

### PR DESCRIPTION
**Pull request by artemkovalyov**
## Context

Minor docs update for the type-safe client release

### CODE REVIEW

File: docs/js/release-notes.mdx [hunk 1]
```diff
@@ -46,21 +46,14 @@ For your convenience we [pre-generate type-safe clients libraries](https://www.n
...
-    - In service `SupplierInvoiceIntegrationService` the methods `createSuplrInvcItemAcctAssgmt` and `createSuplrInvcItemPurOrdRef` have been removed.
-  - For reference on what OData services are included in current release, check out [SAP API Business Hub for SAP S/4HANA Cloud](https://api.sap.com/shell/discover/contentpackage/SAPS4HANACloud)
-
-### Compatibility notes
-
-- Despite releasing a `Beta` version of OData v4 type-safe client we didn't include pre-generated libraries for available S/4HANA OData v4 services in this release. We work to reach `General Availability` for OData v4 client soon and release a type-safe client for OData v4 services together with this milestone. We'll communicate it in the release notes.
-- We recommend updating to SAP Cloud SDK version 1.26.1 to benefit from the latest functionality of type-safe clients.
+  - The update covers all the changes to existing services and introduces new ones. You'll find them on [NPMJS](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*)
+
+## Version 1.21.0 - November 5, 2020
```
Comment (jjtang1985) [inline docs/js/release-notes.mdx:51]: The vdm release is not done, so the date might be changed.

Comment (marikaner) [reply docs/js/release-notes.mdx:51]: Version 1.21.0 ist the old version and was released 2 months ago. Don't have the exact date in mind.

File: docs/js/release-notes.mdx [hunk 2]
```diff
@@ -46,21 +46,14 @@ For your convenience we [pre-generate type-safe clients libraries](https://www.n
...
+
+- [API documentation]()
+- [NPM](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*)
+
+### New functionality
```
Comment (marikaner) [inline docs/js/release-notes.mdx:56]: The latest version should be on top, as usual. ;)

**Comment by artemkovalyov** [comment]
Ooops... I missed this one, but thanks for reviewing. Should have put the docs label on it.
I'll close this as I pushed an update to remove Java-related stuff. I'll update the date when the client is released at last.

**Pull request closed without merging**